stdlib/typing.pyi

```diff
@@ -1180,6 +1182,8 @@ if sys.version_info >= (3, 11):
     def reveal_type(__obj: _T) -> _T: ...
     def assert_never(__arg: Never) -> Never: ...
     def assert_type(__val: _T, __typ: Any) -> _T: ...
+    def clear_overloads() -> None: ...
+    def get_overloads(func: Callable[..., object]) -> Sequence[Callable[..., object]]: ...
```

Comment:
This is an interesting one. I wonder if this is the best return annotation. You *could* call the overload implementations, but it wouldn't be very useful to do so, since all their bodies are empty.

I don't have anything better, though.

Reply:
I guess we could use types.FunctionType. But `inspect.signature` is annotated as taking a Callable, and `inspect.signature` is the main useful thing you could do with the return value of `get_overloads()`.

Reply:
Yeah, that was my basic thought process as well